### DISCUSSION TITLE
How to see the List of Parameters of a Micropython Function on Repl level

### DISCUSSION THREAD

**Discussion by hasenradball** [micropython/micropython#12439] [General]
Hi,

I am working with python since a while, but I have an short question regarding micropython.

If I import a module, which I don't know actually how can I check which parameters has a function inside this module?
With `dir(module)` I get only the function names but not which parameters has a function, right?

**Comment by jimmo** [comment]
MicroPython does not make this available. This is mostly for size/efficiency reasons, but also for some functions (e.g. built-ins implemented in C) the information doesn't exist at all (for example, the parameters of most built-in functions do not actually have names).

**Comment by hasenradball** [comment]
@jimmo Thanks for the fast reply.
So in tis case I have to search for the documentation, right?

**Reply by jimmo** [reply]
Yes.

You could also try setting up stub files for your editor/IDE. https://github.com/Josverl/micropython-stubs

(Note: we're working on adding these stub files to the main repo and making them the source for the documentation).
